core/api/src/commonMain/kotlin/com/livefast/eattrash/raccoonforlemmy/core/api/provider/DefaultServiceProvider.kt

```diff
@@ -31,34 +40,24 @@ internal class DefaultServiceProvider(
     }
 
     override var currentInstance: String = DEFAULT_INSTANCE
-        private set
```

Comment:
when a `val` (in the interface) is overridden by a `var` in the implementation, it is implicitly `private set`.